lib/std/crypto/aes.zig

```diff
@@ -215,6 +240,29 @@ fn AESDecrypt(comptime keysize: usize) type {
         pub fn decrypt(ctx: Self, dst: []u8, src: []const u8) void {
             decryptBlock(ctx.dec[0..], dst, src);
         }
+
+        pub fn ige(ctx: Self, dst: []u8, src: []const u8, iv: [(keysize / 8) * 2]u8) void {
+            std.debug.assert(src.len >= dst.len);
```

Comment:
should this be `==`?

Reply:
I actually think it's the "encrypt" variant that's wrong. Technically I believe you should be able to have a `dst` that's bigger than `src`, but not vice-versa.

Reply:
Actually this is wrong too because the conditions should be reversed